src/common.ts

```diff
@@ -57,3 +59,9 @@ export function filterEmptyRecordValues<TValue>(record: Record<string, TValue |
   }
   return result;
 }
+
+export function resolvePath(cwd: string, arg: string) {
+  return path.isAbsolute(arg)
```

Comment:
Oh, I just assumed path.join handled this by default. Thanks!

Reply:
No problem.  I also hoped that `cd` would take care of `~` and I was tempted to also include that in here -- but I was not sure if you wanted to add the dependency on `std/node/os` or not, so I handled it on my side.  Are you open to the idea for the `cd` PR?

Reply:
That would be good, but I think std/node/os requires an `--unstable` flag to work. We could just copy this function into this library: https://deno.land/std@0.149.0/node/os.ts?code#L161